cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/DiffController.java

```diff
@@ -68,4 +72,45 @@ public Integer call() throws Exception {
             return 0;
         }
     }
+
+    @Command(name = "space-developers", description = "Print the differences between " +
+            "the space developers given in the yaml file and the space developers of your cf instance.")
+    static class DiffSpaceDevelopersCommand implements Callable<Integer> {
+
+        @Mixin
+        LoginCommandOptions loginOptions;
+
+        @Mixin
+        YamlCommandOptions yamlCommandOptions;
+
+        @Override
+        public Integer call() throws Exception {
+            Log.info("Diffing space-developer(s)...");
+            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+            SpaceDevelopersOperations spaceDevOperations = new SpaceDevelopersOperations(cfOperations);
+
+            SpecBean specBeanDesired =
+                    YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
+            specBeanDesired.setApps(null);
+            specBeanDesired.setServices(null);
```

Comment:
Instead of setting all other attributes to null, we could create a new SpecBean and add the space developers from the loaded bean to it. That way, we cannot forget to set an attribute to null (in case we ever extend the SpecBean class to have more attributes than just apps, services and space developers).
We also could offer new constructors in SpecBean that would only take a single attribute as parameter and automatically leave the rest of the other attributes null.